Extensions/ArtifactEngine/package.json

```diff
@@ -18,23 +18,23 @@
   "dependencies": {
     "handlebars": "4.0.14",
     "minimatch": "3.0.2",
-    "azure-pipelines-task-lib": "2.8.0",
+    "azure-pipelines-task-lib": "^3.0.6-preview.1",
     "tunnel": "0.0.4"
   },
   "devDependencies": {
-    "@types/handlebars": "^3.0.28",
+    "@types/handlebars": "^3.0.3",
```

Comment:
Why do we need to change this?

Reply:
Truly speaking bumped it by mistake, since it will be better to remove this types reference. As I could see it is marked in [npm](https://www.npmjs.com/package/@types/handlebars) as deprecated because [handlebars](https://www.npmjs.com/package/handlebars) is already contains in-build type definitions. 
So I will remove this reference.